### ISSUE TITLE
Recommend consolidation

### ISSUE THREAD

**Issue by mchelen-gov** [department-of-veterans-affairs/va.gov-team#41314]
### Description
Recommend consolidating https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html and https://depo-platform-documentation.scrollhelp.site/getting-started/Guidance-for-new-teams-and-team-members.2132803700.html as content is very similar.

### Relevant URLs
https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html
https://depo-platform-documentation.scrollhelp.site/getting-started/Guidance-for-new-teams-and-team-members.2132803700.html

**Comment by lianilychee** [comment]
Thanks, Mike Chelen!

@shiragoodman , sounds like something Governance would handle.  I'll add you as assignee so you can refine in your backlog as you see fit :blob-thumbs-up:

**Comment by shiragoodman** [comment]
On hold as of 5/27. Waiting on @mchelen-gov's feedback

**Comment by shiragoodman** [comment]
thanks @johnny-jesensky-adhoc ! Yes, we intend to keep these pages separate. Happy to discuss this further if you like. Mike Chelen is also no longer our PO and no longer on Platform so I'm not certain if he will have a firm opinion.

**Comment by johnny-jesensky-adhoc** [comment]
Thanks @shiragoodman  - will close this out then.
